src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/workflow.html.twig

```diff
@@ -1,5 +1,127 @@
 {% extends '@WebProfiler/Profiler/layout.html.twig' %}
 
+{% block stylesheets %}
+    {{ parent() }}
+    <style>
+        :root {
+            --bg: rgb(26, 27, 30);
+            --color: #ffffff;
+            --button-bg: rgb(50, 52, 61);
+            --button-color: #ffffff;
+            --warn-bg: rgb(198, 48, 48);
+            --modal-color: rgb(155, 155, 160);
```

Comment:
why do we have dedicated colors there instead of relying on the color variables of the main profiler styles (which would then have support for the dark and light themes)

Reply:
I "backported" something I had in another app, outside the profiler. I'll try to reduce this style

Reply:
I removed many lines of CSS!